internal-api/src/main/kotlin/ua/anastasiia/finesapp/NatsSubject.kt

```diff
@@ -0,0 +1,49 @@
+package ua.anastasiia.finesapp
+
+object NatsSubject {
+    private const val INPUT_PORT = "input"
+    private const val OUTPUT_PORT = "output"
+    private const val REQREPLY_COMMUNICATION_TYPE = "reqreply"
+    private const val PUBSUB_COMMUNICATION_TYPE = "pubsub"
```

Comment:
since these constants are used only one time, you can omit them and use values instead of constants in your inputReqReply and outputPubSub methods